docs/package_index_json-specification.md

```diff
@@ -25,56 +25,56 @@ The root of the JSON index is an array of `packages`:
 
 ```json
 {
-  "packages": [
-    PACKAGE_XXXX
-  ]
+  "packages": [PACKAGE_XXXX]
 }
 ```
 
 3rd party vendors should use a single `PACKAGE_XXXX` that is a dictionary map with the vendor's metadata, a list of `PLATFORMS` and a list of `TOOLS`. For example:
 
+<!-- prettier-ignore -->
 ```json
-    {
-      "name": "arduino",
-      "maintainer": "Arduino LLC",
-      "websiteURL": "http://www.arduino.cc/",
-      "email": "packages@arduino.cc",
-
-      "platforms": [
-        PLATFORM_AVR,
-        PLATFORM_ARM,
-        PLATFORM_XXXXX,
-        PLATFORM_YYYYY,
-      ],
+{
+  "name": "arduino",
+  "maintainer": "Arduino LLC",
+  "websiteURL": "http://www.arduino.cc/",
+  "email": "packages@arduino.cc",
+
+  "platforms": [
+      PLATFORM_AVR,
+      PLATFORM_ARM,
+      PLATFORM_XXXXX,
+      PLATFORM_YYYYY,
+  ],
+
+  "tools": [
+      TOOLS_COMPILER_AVR,
+      TOOLS_UPLOADER_AVR,
+      TOOLS_COMPILER_ARM,
+      TOOLS_XXXXXXX,
+      TOOLS_YYYYYYY,
+   ],
+}
```

Comment:
```suggestion
    {
      "name": "arduino",
      "maintainer": "Arduino LLC",
      "websiteURL": "http://www.arduino.cc/",
      "email": "packages@arduino.cc",

      "platforms": [PLATFORM_AVR, PLATFORM_ARM, PLATFORM_XXXXX, PLATFORM_YYYYY],

      "tools": [
        TOOLS_COMPILER_AVR,
        TOOLS_UPLOADER_AVR,
        TOOLS_COMPILER_ARM,
        TOOLS_XXXXXXX,
        TOOLS_YYYYYYY
      ]
    }
```
The `<!-- prettier-ignore -->` worked great, but I think the de-indentation had already been done by the format prior to setting it to ignore.

I went ahead and ran the formatter on the snippet, then re-indented it. This way, even though the snippet will be ignored by the formatter from here on, at least it is currently compliant (other than intentionally having four extra spaces of indentation). Personally, I'm not really a fan of how it puts the whole array of platforms on a single line, but I'm happy to follow the lead of the formatter. I like most of the choices of the Prettier crew.